src/utils/github_updater.rs

```diff
@@ -28,7 +24,7 @@ const GRAPHQL_UPDATE: &str = "query($ids: [ID!]!) {
             description
             stargazerCount
             forkCount
-            issues(states: [OPEN]) { totalCount }
+            issues { totalCount }
```

Comment:
Why did this change?

Reply:
Oh damn! It's an error, reverting it! Great catch, thanks!